apollo/scripts/addDefaultTagDataToFixedTags.js

```diff
@@ -31,7 +31,8 @@ const now = Date.now();
 const oneWeek = 1000 * 60 * 60 * 24 * 7;
 
 const createUserIdAndTimestamps = () => {
-  const userId = createUserId();
+  // const userId = createUserId();
+  const userId = 'testFakeUserdlYyNSl1lTBEwmpJ';
```

Comment:
Suggestion: hard coding here is ok but it would be helpful to choose a valid id.
For example, you may create an account, find your id, and set here.
This would prevent possible exceptions of user not found.

Reply:
I will change this fake userId to the formal firestore test100's userId
<img width="938" alt="截圖 2023-09-26 上午12 22 30" src="https://github.com/CAMPUS-NYCU/CAMPUS-backend/assets/32840701/8368dd20-3fce-4379-9ae7-a89217ae06ec">

Reply:
Resolved in e9640ae1adabfb3a1ef966887f88cc121e22db22